apps/meeting-tools/dev/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: meeting-tools
+spec:
+  entryPoints:
+    - web
```

Comment:
While this does work (and you likely found it from other deployments in the cluster) it will only serve HTTP traffic, and not HTTPS. We ideally would like this to say `websecure` and then add a TLS cert. Don't make this change now though, as we can do it in the future as an improvement.